src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -116,7 +116,7 @@ private void WriteAssert(string stackTrace, string message, string detailMessage
 
         /// <devdoc>
         ///    <para>
-        ///       Writes the output using <see cref="System.Diagnostics.Debug.Write"/>.
+        ///       Writes the output using <see cref="M:System.Diagnostics.Debug.Write(string)"/>.
```

Comment:
❗️ You can't mix C# keywords with fully-expanded type names. This reference needs to be updated to do either of the following:

1. Remove `M:`
2. Change `string` &rarr; `System.String`

Reply:
Indeed, thank you. I will remove M to stay consistent.